code/modules/mod/mod_theme.dm

```diff
@@ -222,13 +220,13 @@
 	armor = list(MELEE = 20, BULLET = 5, LASER = 5, ENERGY = 5, BOMB = 15, RAD = 15, FIRE = INFINITY, ACID = 150)
 
 /datum/mod_theme/advanced
-	name = "advanced"
-	desc = "An advanced version of Cybersun Industries's classic suit, shining with a white, acid and fire resistant polish."
-	extended_desc = "The flagship version of the Cybersun Industrie industrial model, and their latest product. \
-		Combining all the features of their other industrial model suits inside, with blast resistance almost approaching \
-		some EOD suits, the outside has been coated with a white polish rumored to be a corporate secret. \
-		The paint used is almost entirely immune to corrosives, and certainly looks damn fine. \
-		These come pre-installed with magnetic boots, using an advanced system to toggle them on or off as the user walks."
+	name = "'Daedalus' advanced"
+	desc = "A highly advanced successor to the standard engineering modsuit. Completely radiation, fire, and electricity-proof, with improved conventional armor to boot. The paint gleams like freshly fallen snow."
+	extended_desc = "A product of the bloated generosity of several prominent Nanotrasen executives and their desire to upstage Cybersun at their own game, the NA-20 'Daedalus' represents the peak of powered industrial protection. The thick, double-insulated plastitanium composite carapace offers complete protection from most electrical hazards \
+		in addition to enhanced resistance against heavy impacts, mid-grade explosive detonations, and a slight resistance to directed energy blasts. An underlayer of lead overlain with depleted uranium offers unparalleled protection from even \
+		the most intense radiation exposure, whilst the advanced thermal control system and thermally-reflective surface coating allows the suit to withstand full flame engulfment. The built-in magboots also feature advanced predictive algorithms \
+		so that they can activate and deactivate as the user moves their feet, removing the heavy slogging motion caused by standard magboots.This entire package is supported by a combination of custom joint motors and pseudo-muscle bundles \
```

Comment:
```suggestion
		so that they can activate and deactivate as the user moves their feet, removing the heavy slogging motion caused by standard magboots. This entire package is supported by a combination of custom joint motors and pseudo-muscle bundles \
```